design/baremetal-operator/reboot-interface.md

```diff
@@ -138,6 +138,32 @@ The controller automatically removes all annotations with the
 
 - the Host is deprovisioned
 
+The API has been extended to permit either a hard or a soft reboot to
+take place, specified via the reboot annotation and a `mode` parameter. The API
+supports the following syntax:
```

Comment:
How about if we change this to describe the expectations for the value

```suggestion
The annotation value should be a JSON map containing a key `'mode'` with values of `'hard'` or `'soft'` to specify the reboot mode. The default reboot mode is `'soft'`.

These are a few examples of using the reboot API
```

Reply:
Yep, that's fine. Changing.